presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -2,6 +2,38 @@
 Deploying Presto
 ================
 
+.. _requirements:
+
+Requirements
+------------
+
+.. _requirements-linux:
+
+Linux Operating System
+^^^^^^^^^^^^^^^^^^^^^^
+
+* 64-bit required
+* newer release preferred, especially when running on containers
+
+.. _requirements-java:
+
+Java Runtime Environment
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+* 64-bit required
+* version 11 recommended
+* higher versions receive limited testing
+* Azul Zulu used for builds and testing
+* AdoptOpenJDK, Amazon Corretto, Azul Zulu, and Oracle Java are known to be used
```

Comment:
I think it is good to have that list but not sure if this is sufficient or complete